ddl/ddl_api.go

```diff
@@ -385,6 +407,11 @@ func columnDefToCol(ctx sessionctx.Context, offset int, colDef *ast.ColumnDef, o
 				if hasDefaultValue, value, err = checkColumnDefaultValue(ctx, col, value); err != nil {
 					return nil, nil, errors.Trace(err)
 				}
+				value, err = convertTimestampDelfaultValToUTC(ctx, value, col)
+				if err != nil {
+					return nil, nil, errors.Trace(err)
+
+				}
```

Comment:
Could we use `setDefaultValue` instead of line403-417 or other ways to remove duplicate code？

Reply:
great idea.